docs/user/dql/functions.rst

```diff
@@ -3137,17 +3137,17 @@ Description
 
 ``wildcard_query(field_expression, query_expression[, option=<option_value>]*)``
 
-The wildcard_query function maps to the wildcard_query query used in search engine. It returns documents that match provided text in the specified field.
-Supported wildcard characters can be found here: https://opensearch.org/docs/latest/opensearch/query-dsl/term/#wildcards
-SQL wildcard % is converted to * as well as _ to ?. You may include % and _ in the search by escaping with a backslash prefix.
+The `wildcard_query` function maps to the `wildcard_query` query used in search engine. It returns documents that match provided text in the specified field.
```

Comment:
Move entire `wildcard_query` block a bit up - it should be in `RELEVANCE` group

Reply:
Thanks. Addressed.